cypress/integration/manualTests/zenodo.ts

```diff
@@ -0,0 +1,173 @@
+/*
+ *    Copyright 2018 OICR
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+ import { goToTab } from '../../support/commands';
+
+function testID_Alpha() {
```

Comment:
In TypeScript, at least the way we're doing it, you don't typically have underscores in function names.

Also, I don't think the name of the function conveys its meaning very well. It's a random string generator, not a test function.